src/cita/citation.ts

```diff
@@ -249,6 +252,40 @@ class Citation {
 		// I need access to the parent CitationList
 	}
 
+	wikidataSync() {
+		const syncable = this.source.qid && this.target.qid;
+		const oci = this.getOCI("wikidata");
+		if (oci) {
+			if (oci.valid) {
+				this.resolveOCI("wikidata");
+			} else {
+				// oci is invalid, i.e., citing or cited id do not match with
+				// local source or target id
+				Services.prompt.alert(
+					window as mozIDOMWindowProxy,
+					Wikicite.getString("wikicite.oci.mismatch.title"),
+					Wikicite.formatString("wikicite.oci.mismatch.message", [
+						oci.supplierName.charAt(0).toUpperCase() +
+							oci.supplierName.slice(1),
+						oci.idType.toUpperCase(),
+						oci.citingId,
+						oci.citedId,
+					]),
+				);
+			}
+		} else if (syncable) {
+			//props.sourceItem.syncWithWikidata(index);
```

Comment:
I think it'd be better that this still calls the underlying function so you still get the message that it's unsupported, instead of just nothing happening?